src/datasources/asset/components/AssetQueryEditor.tsx

```diff
@@ -146,7 +197,12 @@ const tooltips = {
 
   workingHours: `Calculate utilization for peak or non-peak hours.`,
 
-  peakDays: `Calculate utilization for peak or non-peak days.`
+  peakDays: `Calculate utilization for peak or non-peak days.`,
+
+  calibrationForecast: {
+    groupBy: `Group the calibration forecast by day, week, or month.`,
```

Comment:
I think we should add an information in the tooltip that only one time based grouping is possible.